picard/script/functions.py

```diff
@@ -1348,3 +1348,22 @@ def func_unique(parser, multi, case_sensitive="", separator=MULTI_VALUED_JOINER)
     if not case_sensitive:
         multi_value._multi = list({v.lower(): v for v in multi_value}.values())
     return multi_value.separator.join(sorted(set(multi_value)))
+
+
+@script_function(eval_args=False, documentation=N_(
+    """`$cleanmulti(name,separator="; ")`
+
+Removes empty elements from the multi-value variable `name`.
```

Comment:
It also works with a literal value, so this description isn't totally exact.

Reply:
Description updated to include a note about using a literal as an input.